### ISSUE TITLE
invalid accession even though file is available

### ISSUE THREAD

**Issue by meghanasp21** [ncbi/sra-tools#478]
Hi.

Over the past couple of days, I have been trying to extract fastq/bam files that I have downloaded but am continuously getting the following error when I get to the fastq-dump step:

2021-03-19T01:07:21 fasterq-dump.2.10.5 err: invalid accession

I was able to get sra files through prefetch:

/cellar/users/andreabc/software/sratoolkit.2.10.8-ubuntu64/bin/prefetch --ngc [project key] SRR7120835 

Then,
I tried doing fasterq-dump

/cellar/users/mpagadal/Programs/sratoolkit.2.10.5-ubuntu64/bin/fasterq-dump --ngc /nrnb/users/ncbi/keys/project_key.ngc --split-files /nrnb/users/ncbi/sra/SRR7120835 _dbGaP-26364.sra 

I've tried with and without the ".sra" prefix. I have tried going into the folder and running the command, but I always get the error. Any help would be appreciated.

**Comment by durbrow** [comment]
Is there some reason you are using two different versions of the toolkit, 2.10.8 for `prefetch` and 2.10.5 for `fasterq-dump`? Otherwise, please use the same version for both.

When you ran `prefetch`, it downloaded to the directory you ran it in. If you run `fasterq-dump` in the same directory and with the same accession name (SRR7120835 in this case), it will find it.

**Comment by meghanasp21** [comment]
Hi. 

I switched, so I ran everything with the most recent version of sratoolkit:
sratoolkit.2.11.0-ubuntu64

I think the issue is that when I prefetch, I'm getting this error:
2021-03-19T12:16:06 prefetch.2.11.0 int: empty while validating file within network system module - cannot open remote file: https://gap-download.be-md.ncbi.nlm.nih.gov/sragap/3FFC41D0-5610-4230-A0D9-0D7EC38AA253/SRR390379?pId=16177

Is there another solution than just to retry the prefetch command?

**Comment by durbrow** [comment]
Unfortunately, it is as you surmise, retrying is the best strategy in the short run. The problem is not with the software on your end, it is likely in the networking infrastructure at NCBI, which is not something this team has any control over.

**Comment by klymenko** [comment]
Please wait, I contacted our systems team - I will notify you about the next step.

**Comment by meghanasp21** [comment]
Hi! Just wanted to follow up on this.

**Comment by klymenko** [comment]
@meghanasp21, please retry.

**Comment by meghanasp21** [comment]
Hi. I'm still getting the following error:

2021-03-25T21:52:57 fasterq-dump.2.10.8 err: empty while validating file within network system module - error with https open 'https://gap-download.be-md.ncbi.nlm.nih.gov/sragap/21994DB3-9540-4011-895D-DD117CE2F962/SRR6795327'
2021-03-25T21:52:57 fasterq-dump.2.10.8 err: invalid accession 'SRR6795327'

**Comment by klymenko** [comment]
We are still waiting for fix from Systems.

**Comment by lapointr** [comment]
Please retry your prefetch and fasterq-dump command

**Comment by muhammadharoon786** [comment]
any solution, I am using the prefetch command but getting same result

**Comment by klymenko** [comment]
@muhammadharoon786, please send email to sra-tools@ncbi.nlm.nih.gov

**Comment by klymenko** [comment]
@meghanasp21, did you have success?

**Comment by meghanasp21** [comment]
I was able to get most downloaded. There are still a couple of files that didn't seem to downloading, but I am re-downloading and seeing if it works. I will keep you updated. Thanks!

**Comment by klymenko** [comment]
@meghanasp21, @muhammadharoon786, do you still need help?

**Comment by muhammadharoon786** [comment]
Yes bro..Sent from my Huawei phone-------- Original message --------From: Andrew Klymenko ***@***.***>Date: Fri, Apr 2, 2021, 7:07 PMTo: ncbi/sra-tools ***@***.***>Cc: muhammadharoon786 ***@***.***>, Mention ***@***.***>Subject: Re: [ncbi/sra-tools] invalid accession even though file is available (#478)
@meghanasp21, @muhammadharoon786, do you still need help?

**Comment by klymenko** [comment]
@muhammadharoon786. did you contact sra-tools@ncbi.nlm.nih.gov?

**Comment by meghanasp21** [comment]
Hi. I think for most of the files, I was able to download them.

However, for some files, I am getting the following error:

2021-03-31T22:01:56 fasterq-dump.2.11.0 err: fasterq-dump.c fastdump_csra() checking ouput-file '/cellar/controlled/dbgap-genetic/phs000409_StJudeMedulloblastoma/fastq/SRR347819.fastq' -> RC(rcExe,rcFile,rcPacking,rcName,rcExists)
fasterq-dump quit with error code 3

Based on previous posts, this can occur if there is not enough disk space. However, I checked to make sure there was plenty available. Is there another solution?
